samples/drivers/watchdog/Makefile

```diff
@@ -1,4 +1,12 @@
 BOARD ?= quark_d2000_crb
-CONF_FILE = prj.conf
+
+ifeq ($(BOARD),96b_carbon)
+CONF_FILE ?= prj_$(BOARD).conf
+else
+ifeq ($(BOARD),nucleo_f334r8)
+CONF_FILE ?= prj_$(BOARD).conf
```

Comment:
@erwango inclusion for L4 family done. I will verify if checking SoC series works.